unity3d/Assets/Swrve/Editor/SwrveCommonBuildComponent.cs

```diff
@@ -43,7 +43,7 @@ protected static void BuildAndroid (string fileName, BuildOptions opt, string[]
         UnityEngine.Debug.Log ("[####] Building " + fileName);
         EditorUserBuildSettings.SwitchActiveBuildTarget (BuildTarget.Android);
         PlayerSettings.bundleIdentifier = packageName;
-        SwrveBuildComponent.CorrectApplicationId ();
+        SwrveBuildComponent.AndroidPreBuild ();
```

Comment:
There is a mention of this method on the upgrade notes, do we need to change it? If so it will have to be mentioned in the upgrade guide too.